src/csc439teamllama/cardgame/GolfController.java

```diff
@@ -21,6 +21,14 @@ public void GameStart(){
 //        example of hand/deck access
         view.TitleScreen();
         game = new GolfGameModel(view.GameStartOptions());
+        for (int i = 0; i < game.players.length; i++) {
```

Comment:
This can be replaced with streams to make the code more fluent and readable.

gamePlayers = game.players.stream()
  .map(player -> {
    // create and return GolfPlayerModel()
   }).collect(toList());
game.players = gamePlayers;

(this is a sample comment to explain the code review process within github)